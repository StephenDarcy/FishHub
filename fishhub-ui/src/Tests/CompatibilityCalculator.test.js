import React from "react";
import renderer from "react-test-renderer";
import CompatabilityCalculator from "../Components/CompatibilityCalculator";

it("renders correctly", () => {
  const tree = renderer
    .create(<CompatabilityCalculator></CompatabilityCalculator>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
