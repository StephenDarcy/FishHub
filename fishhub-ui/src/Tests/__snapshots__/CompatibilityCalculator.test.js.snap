// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="col-sm-4"
  />
  <div
    className="col"
    style={
      Object {
        "textAlign": "right",
      }
    }
  >
    <svg
      enableBackground="new 0 0 48 48"
      fill="currentColor"
      height="50"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      version="1"
      viewBox="0 0 48 48"
      width="50"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M24,6C14.1,6,6,14.1,6,24s8.1,18,18,18s18-8.1,18-18S33.9,6,24,6z M24,10c3.1,0,6,1.1,8.4,2.8L12.8,32.4 C11.1,30,10,27.1,10,24C10,16.3,16.3,10,24,10z M24,38c-3.1,0-6-1.1-8.4-2.8l19.6-19.6C36.9,18,38,20.9,38,24C38,31.7,31.7,38,24,38 z"
        fill="#D50000"
      />
    </svg>
  </div>
  <div
    className="col"
  >
    <h3
      style={
        Object {
          "color": "red",
          "paddingTop": 7,
          "textAlign": "left",
          "width": "100%",
        }
      }
    >
      Incompatible
    </h3>
  </div>
  <div
    className="col-sm-4"
  />
</div>
`;
